npm/ux/data.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type ReData = {
+  status: string;
+  jobParams: any;
+  physicalCounts: any;
+  physicalCountsFormatted: any;
+  logicalQubit: any;
+  tfactory: any;
+  errorBudget: any;
+  logicalCounts: any;
+  new: boolean;
+  frontierEntries: FrontierEntry[];
+};
+
+export type FrontierEntry = {
+  logicalQubit: any;
+  tfactory: any;
+  errorBudget: any;
+  physicalCounts: any;
+  physicalCountsFormatted: any;
+};
+
+export function CreateReData(
+  input: ReData,
+  frontierEntryIndex: number | null = null,
```

Comment:
If this doesn't need to be provided, then just mark it optional, i.e. parameter would be declared `frontierEntryIndex?: number` (see https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters)